doc/user_guide/transform.rst

```diff
@@ -656,50 +656,50 @@ labels.
 
 Window Transform
 ~~~~~~~~~~~~~~~~
-The Window transform performs calculations over sorted groups of data objects.
-These calculations including ranking, lead/lag analysis, and aggregates such
-as running sums and averages. Calculated values are written back to the
-input data stream, where they can be referenced in encodings.
-
-For example, consider the following chart showing time spent on various
-activities during a day:
+The window transform performs calculations over sorted groups of data objects.
+These calculations include ranking, lead/lag analysis, and aggregates such as cumulative sums and averages.
+Calculated values are written back to the input data stream, where they can be referenced by encodings.
 
+For example, consider the following cumulative frequency distribution:
 
 .. altair-plot::
-
```

Comment:
Blank line is required here or the docs won't build

Reply:
That's interesting. They build on my machine, but I'll bear this in mind.